.circleci/config.yml

```diff
@@ -571,7 +571,7 @@ jobs:
             export PULL_TAG="${COMMIT}"
             export TAG="<< parameters.tag >>"
             export REPO="networkservicemesh"
-            export CONTAINERS=(nsmd nsmd-k8s nsmdp crossconnect-monitor test-nse vppagent-icmp-responder-nse vppagent-nsc nsm-init monitoring-nsc vppagent-dataplane vppagent-dataplane-dev admission-webhook vppagent-firewall-nse)
+            export CONTAINERS=(nsmd nsmd-k8s nsmdp crossconnect-monitor nsm-init vppagent-dataplane vppagent-dataplane-dev admission-webhook)
```

Comment:
We don't need to export `test-common` and `vpp-test-common` images because they are only used for testing.